scm/etc/scripts/plot_configs/gabls3_test.ini

```diff
@@ -0,0 +1,61 @@
+gmtb_scm_datasets = output_gabls3_SCM_GFS_v15p2/output.nc, 
+gmtb_scm_datasets_labels = GFSv15p2, 
+plot_dir = plots_gabls3/
+obs_file = ''
+obs_compare = False
+plot_ind_datasets = True
+time_series_resample = True
+
+[time_slices]
+  [[total]]
+    start = 2006, 7, 1, 12
+    end = 2006, 7, 2, 12
+
+[time_snapshots]
+
+[plots]
+  [[profiles_mean]]
+    vars = qc, qv, T
+    vars_labels = 'qc', 'qv', 'T'
+    vert_axis = pres_l
+    vert_axis_label = 'p (Pa)'
+    y_inverted = True
+    y_log = False
+    y_min_option = min             #min, max, val (if val, add y_min = float value)
+    y_max_option = max              #min, max, val (if val, add y_max = float value)
+
+  [[profiles_mean_multi]]
+    [[[T_forcing]]]
+      vars = T_force_tend, dT_dt_PBL, dT_dt_deepconv, dT_dt_shalconv, dT_dt_micro, dT_dt_lwrad, dT_dt_swrad
+      vars_labels = 'force', 'PBL', 'Deep Con', 'Shal Con', 'MP', 'LW', 'SW'
+      x_label = 'K/day'
+      
+  [[profiles_instant]]
+
+  [[time_series]]
+    vars = 'pres_s','lhf','shf','T_s','T_soil_1'
+    vars_labels = 'surface pressure','lhf','shf','skin temperature (K)','soil temperature of layer 1 (K)'
+  
+  [[time_series_multi]]
+    [[[T_soil]]]
+      vars = T_soil_1, T_soil_2, T_soil_3, T_soil_4
```

Comment:
I see, this is also hardcoded to Noah LSM.

Reply:
These plotting configuration files serve as an example for users to modify and/or for me during testing. All of the SCM output and plotting is basically hard-coded at this point, unfortunately.